packages/eds-core-react/src/components/Button/Button.docs.mdx

```diff
@@ -75,3 +75,15 @@ Example All button combinations.
 
 Compact `Button` using `EdsProvider`.
 <Story id="inputs-button--compact" />
+
+### Using the 'as' prop to render as another component
+
+The `as` prop allows you to render the button as another html element or even react component. 
+This is useful if you want to use the button as as a navigation component for a third party routing library such as rect-router.
+
+```tsx
+import { Link } from "react-router-dom"
+import { Button } from '@equinor/eds-core-react'
+
+<Button as={Link} to="/about">About</Button>
```

Comment:
Suggest adding a example with element also, `<Button as="a" to="/about">About</Button>`

Reply:
This would not be possible ( 'to' does not exist on 'a') but i can add an example of 'a' with 'href' and 'target' and 'rel'

Reply:
I expanded the story a little with a normal 'a'

Reply:
Nice!